### PR TITLE
RD-2838 rekoved auth

### PR DESCRIPTION
**Pull request by ghost**
We are adding additional property to the model to indicate when the auth token was revoked.
Refreshing the token means we immediately mark it as not-revoked.

### CODE REVIEW

File: compliant_social_django/__init__.py [hunk 1]
```diff
@@ -1,4 +1,4 @@
-__version__ = '4.3.3'
+__version__ = "4.4.1"
```
Comment (smee30) [inline compliant_social_django/__init__.py]: Should be `4.4.0` really

File: compliant_social_django/storage.py [hunk 2]
```diff
@@ -188,9 +188,15 @@ def refresh_token(self, strategy, *args, **kwargs):
             # break the access token and refresh token out of the extra data
             self.actual_access_token = extra_data.pop('access_token', None)
             self.actual_refresh_token = extra_data.pop('refresh_token', self.actual_refresh_token)
-            if self.set_extra_data(extra_data):
+            was_updated = self.set_extra_data(extra_data)
+            if was_updated or self.revoked:
```
Comment (smee30) [inline compliant_social_django/storage.py]: Wait, so if it wasn't updated but it had previously been revoked, we're now reactivating it? Is that right? Should that maybe be an `and`?

**Review by smee30 — CHANGES_REQUESTED** [review]

File: compliant_social_django/__init__.py (hunk 1 above)
Comment (ghost) [reply compliant_social_django/__init__.py]: 👍

File: compliant_social_django/storage.py (hunk 2 above)
Comment (ghost) [reply compliant_social_django/storage.py]: As discussed - we have removed the condition to improve clarity of this code, even if by just a little bit.

**Review by smee30 — APPROVED** [review]

**Pull request merged**